source/contracts/reporting/ReputationToken.sol

```diff
@@ -45,8 +46,14 @@ contract ReputationToken is DelegationTarget, Typed, Initializable, StandardToke
     }
 
     function migrateIn(address _reporter, uint256 _attotokens) public afterInitialized returns (bool) {
-        require(ReputationToken(msg.sender) == universe.getParentUniverse().getReputationToken());
+        IUniverse _parentUniverse = universe.getParentUniverse();
+        require(ReputationToken(msg.sender) == _parentUniverse.getReputationToken());
         balances[_reporter] = balances[_reporter].add(_attotokens);
+        // Only count tokens migrated toward the available to be matched in other universes. The bonus should not be added
+        universe.increaseRepAvailableForExtraBondPayouts(_attotokens);
```

Comment:
If I am understanding the code correctly, the amount of REP available for migration _out_ is stored as a storage variable on the child universe.  Then when someone mints REP in another universe that is deducted.  The problem with this is that if there are 3 competing universes (with activity) then REP migrated to child A should be mintable on _both_ children B and C.  This means that really we should be keeping a `mapping(mint_destination => uint256)`.

If this greatly increases complexity we can probably just put a TODO in and deal with it post-launch.

Reply:
Right now the REP would be mintable in Universe B and C as described if the dispute bond was against universe A.

The issue did mention possibly allowing the user to specify other universes to attempt and pull from, which isn't incredibly difficult but this seemed like a fair starting point. Pulling automatically would require a bit more work since we'd have to do some iterative logic and take from each pool equally.

My opinion is it's puntable

Reply:
> Right now the REP would be mintable in Universe B and C as described if the dispute bond was against universe A.

Hmm, I'm not seeing this.  Scenario:
1. round1 has tentative winning outcome A, it is disputed (dispute against A) with 100 REP dispute bond
1. round2 has tentative winning outcome A (again), it is disputed (dispute against A) with 1000 REP dispute bond
1. During fork, universe A has 100 REP
1. round1 dispute holder migrates to universe B, pulling REP from A.  They should get 200 REP in universe B.
1. round2 dispute holder migrates to universe C, pulling REP from A.  They should get 1100 REP in universe C.

Since the dispute holders are migrating to different universes, they can both pull (separately) from the REP migrated to universe A, meaning they can each extract up to 100 REP from universe A, not a combined total of 100 REP extracted from universe A.

Reply:
Each bond pull would decrease the amount available to match for minting from Universe A proportional to the amount needed by the bond.

So in the above scenario the round1 bond holder would actually mint 100 * 100/1200 = 8.3 REP
and since there are only 2 bonds in this universe the round2 bond holder would mint what remains.